// Bug happens on JVM , JVM -Xuse-ir
// FILE: tmp0.kt


import kotlin.collections.*
import kotlin.contracts.*
import kotlin.coroutines.*
import kotlin.time.*
import kotlin.test.*
class Outer {
    class Nested{
        fun foo(s: String):String = TODO()
    }

    companion object {
        private fun String.extension(): String = TODO()
    }
}

fun box(): String =TODO()







public interface Aor <T: UByte> : Iterable<Long>, Asserter{

override fun iterator(): kotlin.collections.Iterator<kotlin.Long> = TODO()

override fun assertSame(message: kotlin.String?, expected: kotlin.Any?, actual: kotlin.Any?): kotlin.Unit = TODO()

override fun assertTrue(message: kotlin.String?, actual: kotlin.Boolean): kotlin.Unit = TODO()

override fun fail(message: kotlin.String?, cause: kotlin.Throwable?): kotlin.Nothing = TODO()

var avaz: T 

val Outer.vtfp: T 
get() = TODO()


var hrfe: Set<T> 

var String.jvui: UShort 
get() = TODO()
set(value) = TODO()

var mmgg: Short 

val nvne: UByte 

var HashSet<UInt>.mbvj: Outer 
get() = TODO()
set(value) = TODO()

val Outer.rzrs: LinkedHashMap<UInt, T> 
get() = TODO()


var Long.uckx: Outer 
get() = TODO()
set(value) = TODO()

}


